browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -30,14 +30,26 @@ using content::ResourceType;
 
 namespace brave {
 
+std::string GetGoogleAnalyticsPolyfillJS() {
+  static std::string base64_output;
+  if (base64_output.length() != 0)  {
+    return base64_output;
+  }
+  std::string str = ui::ResourceBundle::GetSharedInstance().GetRawDataResource(
+      IDR_BRAVE_GOOGLE_ANALYTICS_POLYFILL).as_string();
+  Base64UrlEncode(str, base::Base64UrlEncodePolicy::OMIT_PADDING,
+      &base64_output);
+  base64_output = std::string(kJSDataURLPrefix) + base64_output;
+  return base64_output;
+}
+
 std::string GetGoogleTagManagerPolyfillJS() {
   static std::string base64_output;
   if (base64_output.length() != 0)  {
     return base64_output;
   }
   std::string str = ui::ResourceBundle::GetSharedInstance().GetRawDataResource(
       IDR_BRAVE_TAG_MANAGER_POLYFILL).as_string();
-  base64_output.reserve(180);
```

Comment:
these were added as an optimization so the buffer doesn't need to reallocate itself multiple times btw. But it's a micro-optimization so it's ok to get rid of it.